apps/unified_connector/schema.py

```diff
@@ -31,6 +31,13 @@
 )
 
 
+def get_total_source(info):
+    count = ConnectorSourceLead.objects.filter(source__unified_connector__project=info.context.active_project).count()
+    if PP.check_permission(info, PP.Permission.VIEW_UNIFIED_CONNECTOR):
+        return count
+    return
```

Comment:
We don't have to do the count operation if the user doesn't have permission.

```suggestion
def get_total_source(info):
    qs = ConnectorSourceLead.objects.filter(source__unified_connector__project=info.context.active_project)
    if PP.check_permission(info, PP.Permission.VIEW_UNIFIED_CONNECTOR):
        return qs.count()
    return
```